Makefile

```diff
@@ -13,7 +13,8 @@ VERSION ?= $(GIT_VERSION)
 IMAGE ?= $(REPO):$(VERSION)
 BASE_IMAGE ?= public.ecr.aws/eks-distro-build-tooling/eks-distro-minimal-base-nonroot:latest.2
 GOLANG_VERSION ?= $(shell cat .go-version)
-BUILD_IMAGE ?= public.ecr.aws/bitnami/golang:$(GOLANG_VERSION)
+GO_IMAGE_TAG=$(shell cat .go-version | cut -f 1-2 -d".")
```

Comment:
Should we update [.go-version](https://github.com/aws/amazon-vpc-resource-controller-k8s/blob/master/.go-version) file to include the minor version?

Reply:
Updated to Minor version